README.md

```diff
@@ -396,4 +438,6 @@ program. Please do not post security issues to GitHub Issues or any other public
 
 * The conceptual documentation for MSAL Android is available from [Microsoft authentication library for android conceptual documentation](https://aka.ms/msalandroid).
 
+* [FirstLine Worker documentation](TODO://link)

```

Comment:
Once the documentation is published, this should link to https://docs.microsoft.com/azure/active-directory/develop/shared-device-mode.md

Reply:
will update

Reply:
@hamiltonha  I'll keep this change in dev. Please give me a signal when you're ready to merge this to master. Thanks!